Dockerfile

```diff
@@ -22,10 +28,8 @@ RUN clean-install util-linux libsystemd0 bash
 # Avoid symlink of /etc/localtime.
 RUN test -h /etc/localtime && rm -f /etc/localtime && cp /usr/share/zoneinfo/UTC /etc/localtime || true
 
-COPY ./bin/node-problem-detector /node-problem-detector
-
-ARG LOGCOUNTER
-COPY ./bin/health-checker ${LOGCOUNTER} /home/kubernetes/bin/
+COPY ./output/${TARGETOS}_${TARGETARCH}/bin/node-problem-detector /node-problem-detector
+COPY ./output/${TARGETOS}_${TARGETARCH}/bin/health-checker ${LOGCOUNTER} /home/kubernetes/bin/
```

Comment:
AFAICS we're hardcoding `linux` in the rest of the code and not passing in `TARGETOS` as `--build-arg`. Should it maybe be removed? Or am I missing something?

Reply:
ah, sorry, missed the fact that `TARGETOS` is predefined by BuildKit :facepalm: